DOCS-2541: Move component APIs (#3504)
viamrobotics/docs: docs/appendix/apis/components/sensor.md

---
title: "Sensor API"
linkTitle: "Sensor"
weight: 20
type: "docs"

> description: "The sensor API allows you to give commands to your sensor components for getting measurements."

Reviewer: ```suggestion
description: "Commands for getting sensor readings."
```

icon: true
images: ["/icons/components/sensor.svg"]
---

> The sensor API allows you to give commands to your [sensor components](/components/sensor/) for getting measurements.

Reviewer: ```suggestion
The sensor API allows you to get measurements from your [sensor components](/components/sensor/).
```
or
```suggestion
The sensor API allows you to give commands for getting measurements from your [sensor components](/components/sensor/).
```
[opt] I know you're being consistent, but this feels awk and I think this suggestion makes a little more sense.
